Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputModule.cs

```diff
@@ -333,7 +333,7 @@ void IMixedRealitySourceStateHandler.OnSourceLost(SourceStateEventData eventData
                 if (pointer.InputSourceParent == inputSource)
                 {
                     int pointerId = (int)pointer.PointerId;
-                    Debug.Assert(pointerDataToUpdate.ContainsKey(pointerId));
+                   // Debug.Assert(pointerDataToUpdate.ContainsKey(pointerId));
```

Comment:
Accidentally commented out code?

Reply:
Agreed, release code should not contain comments.  Either remove or (in this case)  revert the change.  Asset statements should be retained

Reply:
Related to #6735. Please provide more specific feedback here: Why are you calling a Debug.Assert here? What does this mean to developers? Please provide more descriptive debug information. Otherwise the Editor Console is just filled up with "Assert Failed" errors that are unhelpful and confusing to developers.
![image](https://user-images.githubusercontent.com/32180789/69754114-f02bc680-1109-11ea-8fad-7da2e0bb2603.png)

Reply:
Related to #6735. Please provide more specific feedback here: Why are you calling a Debug.Assert here? What does this mean to developers? Please provide more descriptive debug information. Otherwise the Editor Console is just filled up with "Assert Failed" errors that are unhelpful and confusing to developers.
image
![image](https://user-images.githubusercontent.com/32180789/69758333-16eefa80-1114-11ea-841e-d105128e34e3.png)
😐

Reply:
i'm not sure this needs to be an assertion? it could also be a warning imho.
also i would give more information to the user if we decide to keep this - the current assert log is not providing any information. maybe like: "Trying to remove a pointer in that wasn't registered".
same goes for assert a few lines below: "Trying to remove a pointer that has already been removed"